internal/graph/urnToPrefix.go

```diff
@@ -6,15 +6,22 @@ import (
 )
 
 // URNToPrefix convert  urn string to a valid s3 prefix
-// So urn:gleaner.oih:edmo:00032788b3d1eecf4257bd8ffd42c5d56761a6bf
-// becomes gleaner.oih/[summoned]/edmo/00032788b3d1eecf4257bd8ffd42c5d56761a6bf.jsonld
+// So
+//
+//	urn:gleaner.iow:iow:counties0:1bbde08c130150c48e7d424a60a00d574174bd28
+//
+// becomes
+//
+//	gleaner.iow/summoned/counties0/1bbde08c130150c48e7d424a60a00d574174bd28f.jsonld
 func URNToPrefix(urn, pathelement, suffix string) (string, error) {
 	parts := strings.Split(urn, ":")
 
-	pos := 2 // insert pathelement at this position
-
 	// Use the append function to create a new slice that includes the new element
-	np := append(parts[:pos], append([]string{pathelement}, parts[pos:]...)...)
+	//pos := 3 // insert path element at this position
+	//np := append(parts[:pos], append([]string{pathelement}, parts[pos:]...)...)
+
+	parts[2] = "summoned"
```

Comment:
confused. I thought we were getting rid of the paths.

Reply:
Nice catch.  Only used by the Snip() function but I see my logic is a bit backwards (and doesn't work for the prov objects).   Also, "summoned" is hard coded in Snip() too, However, I think I can resolve this for both paths and remove the hard coding.   I'll fix and push up a new commit.